benchmarks/tracer/scenario.py

```diff
@@ -0,0 +1,44 @@
+import pyperf
+
+from ddtrace import tracer
+
+
+VARIANTS = [{}]
+
+
+def time_trace(loops, variant):
+    range_it = range(loops)
+    t0 = pyperf.perf_counter()
+    for _ in range_it:
```

Comment:
```suggestion
    for _ in range(loops):
```

Reply:
This is a pattern we see in `pyperf` examples which I suppose has to do with avoiding including the `range` call in the timing.

Reply:
Tbh, it's not a big issue for the timings we are collecting:

```
$ python -m timeit -s 'range(10)'
50000000 loops, best of 5: 5.63 nsec per loop
```

Reply:
yeah I was thinking that it would be pretty insignificant compared to what we're measuring. I'm ok with it if it was intentional :+1: